test/test_convert.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import pytest
+
+import paml
+import sbol3
+import tyto
+import uml
+import unittest
+
+from paml_convert.autoprotocol.autoprotocol_specialization import AutoprotocolSpecialization
+
+from paml_convert.autoprotocol.transcriptic_api import TranscripticAPI, TranscripticConfig
+from paml.execution_engine import ExecutionEngine
+from paml_convert.markdown.markdown_specialization import MarkdownSpecialization
+
+
+class TestConvert(unittest.TestCase):
+    @pytest.mark.skip(reason="need to put tx_secrets.json credentials on github first")
+    def test_ludox(self):
+        out_dir = "."
```

Comment:
Is this testing markdown, autoprotocol, or both?
It will also need to have a more rigorous check of the values that are produced.
Are the outputs stable in their serialization?

Reply:
Yes, the outputs should (or can be made to be) stable.  I combined them both for my own edification to know that I can output multiple formats.  I need to test the contents though.